subsys/bluetooth/controller/ll_sw/ull_sync.c

```diff
@@ -548,7 +547,7 @@ static inline struct ll_sync_set *sync_acquire(void)
 	return mem_acquire(&sync_free);
 }
 
-static struct ll_sync_set *is_enabled_get(uint16_t handle)
+struct ll_sync_set *ull_sync_is_enabled_get(uint16_t handle)
```

Comment:
Move this function after the definition of `ull_sync_set_get` alongwith other non-static functions.

Reply:
Done